src/Findologic/Request/Handler/FilterHandler.php

```diff
@@ -13,7 +14,6 @@
 class FilterHandler
 {
     protected const FILTER_DELIMITER = '|';
```

Comment:
I am pretty sure if you select multiple filters, they are separated with this character already. Please ensure that when configuring a filter as "multi select" in the filter configuration, that it still works properly.

Also a character like < or > may be chosen, since you can not configure these as filter names in the Shopware backend.

Reply:
I have used the `~` for the unique filter value delimiter. It is in the `FilterValue` class. This one is for the multi-select yes which we are handling already.

Reply:
And we are not modifying the filter name in this story. Just adding the filter name in front of the existing value to make it unique. So the rest of the functionality should still work.